packages/cli/src/templates/project/package-json.ts

```diff
@@ -25,7 +25,9 @@ export const template = `{
     "prettier": "^1.19.1",
     "typescript": "^3.9.3",
     "ts-node": "^8.6.2",
-    "@types/node": "^13.5.1"
+    "@types/node": "^13.5.1",
+    "ttypescript": "1.5.12",
```

Comment:
Instead of using `ttypescript` and having to use the explicit wrapper `ttsc` it's better to use `ts-patch` as it patches your current typescript with support for transformers, so the rest stays the same. Oh, and supports pre-typecheck transformers!